Makefile

```diff
@@ -188,6 +188,14 @@ githooks-commit:
 githooks-push:
 	@if [ ! -e .git/hooks/pre-push ]; then ln -s ../../bin/pre-push .git/hooks/pre-push; fi
 
+# generate a new shrinkwrap
+shrinkwrap:
+	@rm -rf node_modules
```

Comment:
We now check for node versions in 9a7ca35. This is a warning, and won't stop the target, but it should let folks know that something is up. If there's a slight mismatch it should be fine provided they aren't on npm 2.